src/main/java/com/mcmoddev/alt/AdditionalLootTables.java

```diff
@@ -8,89 +9,100 @@
 
 import com.mcmoddev.alt.data.Config;
 import com.mcmoddev.alt.util.ALTFileUtils;
+
+import com.mcmoddev.alt.proxy.ClientProxy;
 import com.mcmoddev.alt.proxy.CommonProxy;
 import com.mcmoddev.alt.api.PluginLoader;
 import com.mcmoddev.alt.commands.ALTDumpCommand;
 
 import net.minecraft.util.ResourceLocation;
-import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.LootTableLoadEvent;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.DistExecutor;
 import net.minecraftforge.fml.common.Mod;
-import net.minecraftforge.fml.common.Mod.EventHandler;
-import net.minecraftforge.fml.common.Mod.Instance;
-import net.minecraftforge.fml.common.SidedProxy;
-import net.minecraftforge.fml.common.event.FMLFingerprintViolationEvent; 
-import net.minecraftforge.fml.common.event.FMLInitializationEvent;
-import net.minecraftforge.fml.common.event.FMLInterModComms;
-import net.minecraftforge.fml.common.event.FMLPostInitializationEvent;
-import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
-import net.minecraftforge.fml.common.event.FMLServerStartingEvent;
+import net.minecraftforge.fml.event.lifecycle.FMLCommonSetupEvent;
+import net.minecraftforge.fml.event.lifecycle.InterModProcessEvent;
+import net.minecraftforge.fml.event.server.FMLServerStartingEvent;
+import net.minecraftforge.fml.event.server.FMLServerStoppedEvent;
+import net.minecraftforge.fml.loading.FMLPaths;
+
 
-@Mod(modid = AdditionalLootTables.MODID,
-	version = AdditionalLootTables.VERSION,
-	name = AdditionalLootTables.NAME,
-	acceptedMinecraftVersions = "[1.12,)",
-	certificateFingerprint = "@FINGERPRINT@")
+@Mod(AdditionalLootTables.MODID)
 public class AdditionalLootTables {
-	public static final String NAME = "Additional Loot Tables";
 	public static final String MODID = "alt";
 	public static final String VERSION = "2.0.3";

```

Comment:
This should probably be incremented as there is a lot of change in the port thanks to major differences in how 1.15 works compared to 1.12. Perhaps 2.1.0 ?